### PR TITLE
Add support for .msix and .msixbundle files

### PR DESCRIPTION
**Pull request by acroos**
The Windows team is adding a new file extension - .msix, .msixbundle, etc. - in this years fall release. It's compatible with .appx and should be handled the same way.

### CODE REVIEW

**Comment by acroos** [comment]
@izikl I'm not able to request a reviewer, but here is the PR we discussed earlier today on slack.

File: README.md [hunk 1]
```diff
@@ -98,16 +98,16 @@ Supported extensions: .appx, appxupload or .zip
 
 ### UWP Bundles
 ```
-Supported extensions: .appxbundle, appxupload or .zip
+Supported extensions: .appxbundle, .appxupload, .msixbundle, .msixupload or .zip
 ```
 
-> For .appxbundle app packages, the manifest is checked for the correct name of the appx subpackage. If it is found, the metadata is parsed directly from there, otherwise we scavenge for as much as we can get from the appxbundle manifest and metadata
+> For .appxbundle or .msixbundle app packages, the manifest is checked for the correct name of the appx subpackage. If it is found, the metadata is parsed directly from there, otherwise we scavenge for as much as we can get from the appxbundle/msixbundle manifest and metadata
```
Comment (izikl) [inline README.md]: According to this page: 
https://mspoweruser.com/microsoft-announces-new-msix-app-packaging-format/
```
This new MSIX installer will support all type of Windows applications including Win32, WPF, WinForm and UWP.
```

Will this change support UWP apps only? If so, can you please document it?

**Comment by izikl** [comment]
Please bump the version in the package.json file

File: README.md (hunk 1 above)
Comment (acroos) [reply README.md]: I'm not sure to be honest.  I'll try to reach out and document this if necessary

**Pull request merged**